app/models/item.rb

```diff
@@ -8,4 +8,12 @@ class Item < ActiveRecord::Base
   has_many :orders, through: :order_items
   has_attached_file :image, :default_url => "/images/default.jpg"
   validates_attachment_content_type :image, :content_type => [ "image/jpg", "image/jpeg", "image/png" ]
+
+  def self.search(search)
+    if search
+      where('title LIKE ?', "%#{search}%") + where('description LIKE ?', "%#{search}%")
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.